src/Fable.Import.Electron.fs

```diff
@@ -22,24 +22,33 @@ module ElectronPrivate =
 
 module Electron =
     type [<Import("EventEmitter", "electron")>] EventEmitter() =
-        interface NodeJS.EventEmitter with
-            member __.addListener(``event``: string, listener: Function): NodeJS.EventEmitter = failwith "JS only"
-            member __.on(``event``: string, listener: Function): NodeJS.EventEmitter = failwith "JS only"
-            member __.once(``event``: string, listener: Function): NodeJS.EventEmitter = failwith "JS only"
-            member __.removeListener(``event``: string, listener: Function): NodeJS.EventEmitter = failwith "JS only"
-            member __.removeAllListeners(?``event``: string): NodeJS.EventEmitter = failwith "JS only"
-            member __.setMaxListeners(n: int): unit = failwith "JS only"
+        interface Node.Events.event_types.EventEmitter with
+            member __.addListener(``event``: U2<string, Symbol>, listener: Function): Node.Events.event_types.EventEmitter = failwith "JS only"
```

Comment:
Maybe it's enough `string` here instead of `U2<string, Symbol>` (same below) as users probably won't be using Symbols from F#.